applications/nrf_desktop/doc/generated/event_rel_modules/ble_peer_event.rst

```diff
@@ -0,0 +1,22 @@
+.. _nrf_desktop_ble_peer_event_sources:
```

Comment:
I'm against using two formatting patterns for representing the module relations.
If we use tables, we should use them consistently everywhere.
If tables are too complicated for some modules, then we need to rethink them.

This is not to be picky: we got feedback that these tables are hard to get, and we added explanation about them. But introducing two formatting standards adds to complexity and goes against this feedback.